Tests don't pass for skcuda backend

Currently, two tests fail on the skcuda backend because they require CPU tensors. Specifically, we get
```
$ KYMATIO_BACKEND=skcuda pytest kymatio --tb=line
============================= test session starts ==============================
platform linux -- Python 3.7.0, pytest-3.9.1, py-1.7.0, pluggy-0.8.0
rootdir: /home/janden/projects/kymatio, inifile:
collected 37 items                                                             

kymatio/scattering1d/tests/test_filters.py ........                      [ 21%]
kymatio/scattering1d/tests/test_scattering1d.py ........F                [ 45%]
kymatio/scattering1d/tests/test_utils.py ....                            [ 56%]
kymatio/scattering2d/tests/test_scattering2d.py ....F                    [ 70%]
kymatio/scattering3d/tests/test_scattering3d.py ...........              [100%]

=================================== FAILURES ===================================
/home/janden/projects/kymatio/kymatio/scattering1d/backend/backend_skcuda.py:152: RuntimeError: Use the torch backend for cpu tensors!
/home/janden/projects/kymatio/kymatio/scattering2d/backend/backend_skcuda.py:281: RuntimeError: Use the torch backend for cpu tensors!
=============================== warnings summary ===============================
/home/janden/local/anaconda3/envs/kymatio/lib/python3.7/site-packages/torch/autograd/gradcheck.py:2: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable

/home/janden/local/anaconda3/envs/kymatio/lib/python3.7/site-packages/torch/nn/modules/container.py:2: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import OrderedDict, Iterable, Mapping

/home/janden/local/anaconda3/envs/kymatio/lib/python3.7/site-packages/_pytest/python.py:166: RuntimeWarning: The skcuda backend does not pass differentiabilitytests, but that's ok (for now).
  testfunction(**testargs)

/home/janden/local/anaconda3/envs/kymatio/lib/python3.7/site-packages/_pytest/python.py:166: RuntimeWarning: The skcuda backend does not pass differentiabilitytests, but that's ok (for now).
  testfunction(**testargs)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=============== 2 failed, 35 passed, 4 warnings in 37.21 seconds ===============
```

--- a/kymatio/scattering1d/tests/test_scattering1d.py
+++ b/kymatio/scattering1d/tests/test_scattering1d.py
@@ -292,6 +292,11 @@ def test_batch_shape_agnostic():
     assert "at least one axis" in ve.value.args[0]
 
     x = torch.zeros(shape)
+
+    if force_gpu:
+        x = x.cuda()
+        S.cuda()
+
     Sx = S(x)
 
     assert Sx.dim() == 2
@@ -303,6 +308,9 @@ def test_batch_shape_agnostic():
 
     for test_shape in test_shapes:
         x = torch.zeros(test_shape)
+
+        if force_gpu:
+            x = x.cuda()
 
         S.vectorize = True
         Sx = S(x)

--- a/kymatio/scattering2d/tests/test_scattering2d.py
+++ b/kymatio/scattering2d/tests/test_scattering2d.py
@@ -234,6 +234,11 @@ def test_batch_shape_agnostic():
     assert "at least two" in ve.value.args[0]
 
     x = torch.zeros(shape)
+
+    if backend.NAME == 'skcuda':
+        x = x.cuda()
+        S.cuda()
+
     Sx = S(x)
 
     assert len(Sx.shape) == 3
@@ -246,6 +251,9 @@ def test_batch_shape_agnostic():
 
     for test_shape in test_shapes:
         x = torch.zeros(test_shape)
+
+        if backend.NAME == 'skcuda':
+            x = x.cuda()
 
         Sx = S(x)
 

